packages/cli/package.json

```diff
@@ -31,6 +31,6 @@
     "strip-ansi": "^6.0.0"
   },
   "engines": {
-    "node": ">=14.18.0"
+    "node": ">=18.18.0"
```

Comment:
I think we could just use `"node": ">=18.0.0"`. 18.18.0 is a weird threshold, it's neither the current latest 18 nor it's the one that when 18 was marked as LTS for the first time (that was 18.12.0).

Reply:
It is weird. I thought we needed that patch version for some reason, but looking at the changelog I'm completely wrong.

I think that we should be fine setting it at `"node": ">=18.0.0"`, I don't see a reason we would need the LTS specifically.